src/buddy/main.cpp

```diff
@@ -330,6 +314,19 @@ void spi_set_prescaler(SPI_HandleTypeDef *hspi, int prescaler_num) {
     HAL_SPI_Init(hspi);
 }
 
+void init_redscreen() {
```

Comment:
Rename function to something like `init_error_screen`. Redscreen is not the only relevant screen here.